docs/source/stylesheets/screen.css.scss

```diff
@@ -213,7 +213,7 @@ html, body {
 #nav-button {
   span {
     display: block;
-    $side-pad: $main-padding / 2 - 8px;
+    $side-pad: $main-padding * 0.5 - 8px;
```

Comment:
You should use `calc` to divide or multiply.
Docs - https://developer.mozilla.org/en-US/docs/Web/CSS/calc

Reply:
I have updated the code to use calc() as suggested, please review the changes.